tests/Behat/FeatureContext.php

```diff
@@ -68,12 +68,12 @@ public function assertSelectOptions(string $select, TableNode $options): void {
    * @Then the current workflow state should be :state
    */
   public function assertCurrentWorkflowState(string $state): void {
-    $element = $this->getSession()->getPage()->find('css', 'div#edit-moderation-state-0-current');
+    $element = $this->getSession()->getPage()->find('css', 'div#edit-current');
     if (empty($element)) {
       throw new \Exception('The current workflow state field is not present on the page.');
     }
     // Need to remove the label from the string.
-    Assert::assertEquals($state, ltrim(trim($element->getText()), 'Current state'));
+    Assert::assertEquals($state, ltrim(trim($element->getText()), 'Moderation state'));
```

Comment:
This is an unsafe usage of `ltrim()`, the second parameter is the character list to remove. In this case it works, but it wouldn't in other cases:
```
>>> ltrim('Moderation state Modified', 'Moderation state');
=> "fied"
```
Anyway with the remark above this part is dropped.